libraries/AP_HAL_ChibiOS/hwdef/BlitzH743Pro/README.md

```diff
@@ -0,0 +1,89 @@
+# iFlight BLITZ H7 Pro Flight Controller
+
+https://shop.iflight-rc.com/BLITZ-Whoop-F7-AIO-Pro1927
```

Comment:
this is not the link.....and the only H& I can find on the iFlight site is a WING autopilot